### ISSUE TITLE
Possible Issue for Django 1.6 in django_pgjson 0.3.0

### ISSUE THREAD

**Issue by themaxxx** [djangonauts/django-pgjson#28]
Hi,

in fields.py there is this import 

"from django.utils.module_loading import import_string"

but in Django 1.6 import_string is not present.

My current solution is :
if django.get_version() >= "1.7":
    from django.utils.module_loading import import_string
else:
    from django.utils.module_loading import import_by_path as import_string

Please let me know your opinion. Thx a lot.

**Comment by niwinz** [comment]
Yeah, seems clearly a compatibility bug. If that solution works as expected, it is ok for me. PR is welcome ;)

**Comment by themaxxx** [comment]
Hi,
I forked the project and fixed the compatibility bug in case you want to pull it into the project repository.
Thanks!

**Comment by sibartlett** [comment]
Yeah, I'm getting this error too.
